src/hints/block_context.rs

```diff
@@ -107,16 +98,17 @@ pub fn load_deprecated_inner(
     let (class_hash, deprecated_class) = deprecated_class_iter.next().unwrap();
 
     exec_scopes.insert_value("compiled_class_hash", class_hash);
+    exec_scopes.insert_value("compiled_class", deprecated_class.clone());
 
     let dep_class_base = vm.add_memory_segment();
     write_deprecated_class(vm, dep_class_base, deprecated_class)?;
+    println!("{dep_class_base:}");
 
     insert_value_from_var_name("compiled_class", dep_class_base, vm, ids_data, ap_tracking)
 }
 
-/// Implements hint:
-///
-/// memory[ap] = to_felt_or_relocatable(deprecated_syscall_handler.block_info.block_number)
+pub const DEPRECATED_BLOCK_NUMBER: &str =
+    indoc! {"memory[ap] = to_felt_or_relocatable(deprecated_syscall_handler.block_info.block_number)"};
```

Comment:
There is no need to indoc one-liners.

Reply:
this will have the same functionality?:

```
pub const DEPRECATED_BLOCK_NUMBER: &str = "memory[ap] = to_felt_or_relocatable(deprecated_syscall_handler.block_info.block_number)";
```

Reply:
indoc is taken out for one-liners